pkg/chartverifier/certificate.go

```diff
@@ -16,76 +16,88 @@
 
 package chartverifier
 
-import "strconv"
+import (
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	helmchart "helm.sh/helm/v3/pkg/chart"
```

Comment:
is it really needed to introduce package alias? IMHO `chart.Foo` reads perfectly well.

Reply:
I find "chart" too general, as this is actually the helm chart and I think the alias makes the code more clear.

Reply:
ok